test/functional/fixtures/api/es-next/selector/testcafe-fixtures/selector-test.js

```diff
@@ -15,6 +15,34 @@ const isIEFunction = ClientFunction(() => {
            appName === 'Netscape' && isIE11Re.exec(userAgent) !== null;
 });
 
+
```

Comment:
Let's place this tests near other `filter` tests in the file (for example after the `Combination of filter methods` test).